mesonbuild/backend/vs2010backend.py

```diff
@@ -1437,6 +1489,12 @@ def path_normalize_add(path, lis):
                     self.add_include_dirs(lang, inc_cl, inc_dirs)
                     # XXX: Do we need to set the object file name name here too?
 
+        # Filter information
+        filter_group = ET.SubElement(root_filter, 'ItemGroup')
+        for filter_dir in list_filters_path:
+            filter = ET.SubElement(filter_group, 'Filter', Include=filter_dir)
+            ET.SubElement(filter, 'UniqueIdentifier').text = '{' + str(uuid.uuid1()) + '}'
```

Comment:
I think you should convert the uuid in uppercase. Also, would `uuid4` be better than `uuid1`? Maybe it is not relevant in this context...

Reply:
Hello Charles,
Thanks for the quick review.
Default uuid are generated in uppercase when you create a project from scratch but, uuid are in lowercase if you create manually a filter inside Visual Studio.
I think it doesn't matter much for Visual Studio.

Yes uuid4 seems to be more appropriate than uuid1. Thanks.
I'll make a fix.

Reply:
If you want I'll put in uppercase the uuid. (It is working without)